src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -1033,7 +1033,7 @@ public Role createRole(GuildImpl guild, DataObject roleJson, long guildId)
         final int color = roleJson.getInt("color");
         role.setName(roleJson.getString("name"))
             .setRawPosition(roleJson.getInt("position"))
-            .setRawPermissions(roleJson.getLong("permissions"))
+            .setRawPermissions(roleJson.getLong("permissions_new"))
```

Comment:
Wow you're fast.

FYI you might need to do `Long.parseLong(roleJson.getString("permissions_new"))` here, some libraries don't automatically cast the string to a long.

Reply:
Don't worry this is a higher-level abstraction which does the type coalescing for me already.